build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+allprojects {
+    group = 'RoboViz'
+    version = '0.0.1-SNAPSHOT'
```

Comment:
We should probably use the actual RoboViz version here? :)

Reply:
resolved in f221c91